src/main/java/vimification/internal/parser/ArgumentFlag.java

```diff
@@ -0,0 +1,55 @@
+package vimification.internal.parser;
+
+import java.util.Objects;
+
```

Comment:
I think this is a good abstraction. @DerenC @liujiayue314 we might have to take note of this and write down all the `longForm` and `shortForm` for @VietAnh1010 to keep track of.